### PR TITLE
update models

### PR DESCRIPTION
**Pull request by stevenbuschbach**
most recent version of steve's code.

### CODE REVIEW

**Comment by satejsoman** [comment]
will take a look at this in more depth tomorrow but looks good on first inspection!

File: studies/us_states_validation/etl.py [hunk 1]
```diff
@@ -32,26 +32,79 @@ def import_and_clean_cases(save_path: Path) -> pd.DataFrame:
...
 
-def get_rt_live_data(save_path: Path) -> pd.DataFrame:
-    '''
-    Gets Rt estimates from Rt.live.
-    '''
+def get_adaptive_estimates(path: Path) -> pd.DataFrame:
+    
+    # Parameters for filtering raw df
+    kept_columns   = ['date','state','RR_pred','RR_CI_lower','RR_CI_upper','T_pred',
+                      'T_CI_lower','T_CI_upper','new_cases_ts','anamoly']
```
Comment (satejsoman) [inline studies/us_states_validation/etl.py:39]: i hate being a pedant but `anomaly`

File: studies/us_states_validation/cori_model.R [hunk 2]
```diff
@@ -0,0 +1,98 @@
+# Set working directory to current file path
```
Comment (satejsoman) [inline studies/us_states_validation/cori_model.R:1]: is this file straight from the cobey lab repo?

File: studies/us_states_validation/us_states_rt_est.py [hunk 3]
```diff
@@ -116,16 +181,33 @@ def make_state_plots(df: pd.DataFrame, plotspath: Path) -> None:
...
+    # Get data case data
+    df = import_and_clean_cases(data)
+
+    # Run models for adaptive and rt.live old version
+    # run_adaptive_model(df=df, CI=CI, smoothing=convolution(window=smoothing_window), filepath=data)
+    # run_rtlive_old_model(df=df, filepath=data)
+    # run_luis_model(df=df, filepath=data)
+    # run_cori_model(filepath=root, rexepath=rexepath)
+
+    # Pull CSVs of results
```
Comment (satejsoman) [inline studies/us_states_validation/us_states_rt_est.py:193]: this is so great :)

File: studies/us_states_validation/etl.py (hunk 1 above)
Comment (stevenbuschbach) [reply studies/us_states_validation/etl.py:39]: Oof yeah, I kept messing up that spelling. Good catch.

File: studies/us_states_validation/cori_model.R (hunk 2 above)
Comment (stevenbuschbach) [reply studies/us_states_validation/cori_model.R:1]: It is ~80% from the Cobey lab repo, with some of my adjustments to adapt it to our use case. I meant to add a comment with a link attributing it to them. Will include that in a future iteration.

**Pull request merged**